core/lib/spree/core/controller_helpers/order.rb

```diff
@@ -21,10 +21,10 @@ def simple_current_order
           @simple_current_order = find_order_by_token_or_user
 
           if @simple_current_order
-            @simple_current_order.last_ip_address = ip_address
+            @simple_current_order.update(last_ip_address: ip_address)
```

Comment:
The test in the fixup looks good to me (it can't hurt to have). I'd like it rebased please :)